agentstack/templates/autogen/{{cookiecutter.project_metadata.project_slug}}/src/main.py

```diff
@@ -14,7 +14,7 @@ def main():
     # For example, if you have created a OAI_CONFIG_LIST file in the current working directory, that file will be used.
     config_list = config_list_from_json(env_or_file="OAI_CONFIG_LIST")
 
-    {%- for agent in cookiecutter.structure.agents %}
```

Comment:
please disable your formatter for this project. the additional whitespaces here will break cookiecutter